docs/gallery/interactive/sofar_introduction.ipynb

```diff
@@ -0,0 +1,369 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# sofar and SOFA\n",
+    "\n",
+    "If you are new to SOFA and/or sofar, this is a good place to start. SOFA is short for *Spatially Oriented Format for Acoustics* and is an open file format for saving acoustic data, for example head-related impulse responses (HRIRs). Good places to get more information about SOFA after this introduction are\n",
+    "\n",
+    "* The [SOFA paper](https://doi.org/10.17743/jaes.2022.0026)\n",
+    "* The documentation of the [SOFA conventions](https://sofar.readthedocs.io/en/stable/resources/conventions.html)\n",
+    "* [sofaconventions.org](https://www.sofaconventions.org)\n",
+    "* The SOFA standard [AES69-2022](https://www.aes.org/publications/standards/search.cfm?docID=99)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Creating SOFA objects\n",
+    "\n",
+    "To cover a variety of data, SOFA offers different *conventions*. A convention defines, what data can be saved and how it is saved. You should always find the most specific convention for your data. This will help you to identify relevant data and meta data that you should provide along the actual acoustic data. Using sofar, a list of possible conventions can be obtained with"
```

Comment:
maybe mention the main difference first, like TF, IR and SOS and then go the msot specific case.

Reply:
Good point. I added more information on that.